### PR TITLE
feat: new ui for mnemonic onboarding

### LINKED ISSUE #2926: Update Master Key back up screens
Part of https://github.com/getAlby/lightning-browser-extension/issues/2917

### LINKED ISSUE #2932: Update Master Key back up and import flows
Part of #2917 epic

For detailed inspect of designs, [see this Figma file](https://www.figma.com/file/IvlUHnC3vkIWnQ14FppZqT/%F0%9F%8E%A8-Extension-UI-revamp?type=design&node-id=877-18928&mode=design&t=7prPviaZBt6uBX6l-4)

### To do

- [ ] Update copy on both screens with new [Popicons](https://github.com/getAlby/lightning-browser-extension/issues/2920)
- [ ] Illustration update (light and dark mode) and it goes below copy
- [ ] Add border 1px inner gray200 / dark: neutral700
- [ ] Change corner radius to rounded-2xl
- [ ] Put buttons inside the card, center and fix 256px (w-64)
- [ ] Cards should have p-10 padding (40px)
- [ ] Ideally, recovery phrase box with 12 words is narrowed and layout follows the one in Figma (doesn't have to be ideally the same though)

### New illustrations:
Type=Dark
Type=Light

Custom icon (not available in Popicons):
face-surprise

### Screenshots

image

image

image

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

ui improvements for mnemonic onboarding

### Link this PR to an issue [optional]

Fixes  #2932

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

File: static/assets/images/master_key_dark.png
Comment (reneaaron) [inline static/assets/images/master_key_dark.png:1]: Image needs to be at least in 2x of the display size for retina displays. The new image looks very blurry on my screen.

**Review by reneaaron — COMMENTED** [review]
When there is no key yet and I click on "Generate new Master Key" I don't see the 3 list items explaining the backup of the Master Key:

image

They are currently only being shown when visiting the backup page (after generating). I think it would be useful to add these 3 points there as well. 
 
image

Most users probably will never visit the backup page (as they confirm the backup while generating the key). cc @stackingsaunter

File: src/app/components/mnemonic/MnemonicBackupDescription/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  title: string;
+  type: "warn" | "info";
+};
+
+function ListItem({ icon, title, type }: ListItemProps) {
+  return (
+    <div className="flex gap-2 items-center">
+      <div
+        className={classNames(
+          type == "warn" && "text-orange-700 dark:text-orange-200",
```
Comment (reneaaron) [inline src/app/components/mnemonic/MnemonicBackupDescription/index.tsx:50]: Apply colors on outer element to avoid duplication.

**Comment by stackingsaunter** [comment]
@pavanjoshi914 here you have 2x illustrations in png and svg 

Master Key Dark
Master Key Dark
Master Key Light
Master Key Light

**Comment by pavanjoshi914** [comment]
@reneaaron we still need the designs for generating the master key page. @stackingsaunter is working on it i think!

**Comment by stackingsaunter** [comment]
CleanShot 2024-01-18 at 12 10 05@2x

This icon is a png, here's svg:

face-surprise

There's also a copy duplication: 
CleanShot 2024-01-18 at 12 12 02@2x

**Comment by stackingsaunter** [comment]
@pavanjoshi914 


you meant this right? (when user deletes their master key)
I'd like settings to look like this

## User, when after deleting Master Key
image


The generation itself would look the same (and could happen after users click Continue on explenation page?)

I posted designs [in the file as well](https://www.figma.com/file/IvlUHnC3vkIWnQ14FppZqT/%F0%9F%8E%A8-Extension-UI-revamp?type=design&node-id=1240-23743&mode=design&t=fFnH4pDsRBWZJtAJ-4)

Other screens:

## User with MK 
image


## User with MK, but imported Nostr wallet
image

File: src/app/components/mnemonic/MnemonicBackupDescription/index.tsx (hunk 1 above)
Comment (pavanjoshi914) [reply src/app/components/mnemonic/MnemonicBackupDescription/index.tsx:50]: resolved

**Comment by pavanjoshi914** [comment]
@reneaaron there is separate pr for default master key generation. with this user will only see back everytime and generate master key screen will be only seen when user delete existing key and want to generate new key. https://github.com/getAlby/lightning-browser-extension/pull/2988

shall we keep generate screen same as backup screen @stackingsaunter ? we don't have desings for this screen yet

**Comment by stackingsaunter** [comment]
In my opinion yes, what would you change? @pavanjoshi914 
What;s your opinion @reneaaron ?

**Comment by pavanjoshi914** [comment]
image
image

**Comment by stackingsaunter** [comment]
I though the goal was to autogenerate the key, does it make sense to make it separate from UI then? In this PR there's no autogeneration cc @reneaaron @pavanjoshi914 

Some feedback:
- As we discussed on Slack, it's better to revert to white bg in text field everywhere (also in seed fields)
- I think we can delete the recovery phrase explanation points in "Import" screen, and keep them only in generate/backup screens
- This is minor, but it would be great if we can have those inputs somehow aligned. If it's too hard then I'd say don't worry about it:


In Figma I did it by defining the width of number in front of the field the same size, so for double digits, and centered the numbers:




- This may be unrelated to this PR and could get another one but I'd not use emojis in the notification bars. And instead use icons:


Maybe for now you can just delete the emoji

**Comment by pavanjoshi914** [comment]
@stackingsaunter autogeneration is here https://github.com/getAlby/lightning-browser-extension/pull/2988 there was separate issue so i created separate pr

File: static/assets/images/face_surprise_dark.png
Comment (reneaaron) [inline static/assets/images/face_surprise_dark.png:1]: Replace these icons with SVGs, use fill=currentColor for light & darkmode.

File: static/assets/icons/popicons/eye-crossed.svg
Comment (reneaaron) [inline static/assets/icons/popicons/eye-crossed.svg:1]: Use an svg optimizer.

File: src/i18n/locales/en/translation.json
Comment (reneaaron) [inline src/i18n/locales/en/translation.json:1]: Run the suggested commands. Those translations should be updated.

File: src/app/components/mnemonic/MnemonicDescription/index.tsx [hunk 2]
```diff
@@ -33,10 +33,10 @@ type ListItemProps = { icon: React.ReactNode; title: string };
 function ListItem({ icon, title }: ListItemProps) {
   return (
     <div className="flex gap-2 items-center">
-      <div className="shrink-0 w-8 h-8 text-gray-600 dark:text-neutral-400">
-        {icon}
-      </div>
-      <span className="text-gray-600 dark:text-neutral-400">{title}</span>
+      <div className="shrink-0 text-gray-600 dark:text-neutral-400">{icon}</div>
```
Comment (reneaaron) [inline src/app/components/mnemonic/MnemonicDescription/index.tsx:36]: move text color classes to parent

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: static/assets/images/face_surprise_dark.png
Comment (reneaaron) [reply static/assets/images/face_surprise_dark.png:1]: @stackingsaunter Can you please provide this as an SVG?

File: static/assets/icons/popicons/eye-crossed.svg
Comment (pavanjoshi914) [reply static/assets/icons/popicons/eye-crossed.svg:1]: resolved

File: src/i18n/locales/en/translation.json
Comment (pavanjoshi914) [reply src/i18n/locales/en/translation.json:1]: resolved

File: static/assets/images/face_surprise_dark.png
Comment (pavanjoshi914) [reply static/assets/images/face_surprise_dark.png:1]: resolved

File: src/i18n/locales/en/translation.json
Comment (pavanjoshi914) [reply src/i18n/locales/en/translation.json:1]: resolved

File: src/app/components/mnemonic/MnemonicDescription/index.tsx (hunk 2 above)
Comment (pavanjoshi914) [reply src/app/components/mnemonic/MnemonicDescription/index.tsx]: resolved

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**